lib/remote_persistent_term/fetcher/s3.ex

```diff
@@ -30,11 +30,12 @@ defmodule RemotePersistentTerm.Fetcher.S3 do
       required: true,
       doc: "The AWS region of the s3 bucket."
     ],
-    failover_regions: [
-      type: {:list, :string},
+    failover_buckets: [
+      type: {:list, :keyword_list},
```

Comment:
can you define a a schema for the failover buckets, so that nimble options validates it has the correct key/values?